edgelet/Makefile

```diff
@@ -194,6 +197,15 @@ install: release
 
 install-rpm: install
 	$(INSTALL_DATA) -D $(rpmbuilddir)/SOURCES/00-aziot-edged.preset $(DESTDIR)$(presetdir)/00-aziot-edged.preset
+	$(INSTALL) -m 0660 /dev/null $(DESTDIR)$(localstatedir)/lib/iotedge/mgmt.sock
+	$(INSTALL) -m 0666 /dev/null $(DESTDIR)$(localstatedir)/lib/iotedge/workload.sock
```

Comment:
You removed these lines from the `install` target above and added them under `install-rpm` here. Is that a problem in the non-rpm case?

Reply:
Yes, Debian packaging makes the `deb` target, `install` is used in the RPM packaging, they shouldn't overlap.